lang/LangSource/SC_LanguageConfig.hpp

```diff
@@ -84,6 +85,7 @@ class SC_LanguageConfig
 	DirVector mIncludedDirectories;
 	DirVector mExcludedDirectories;
 	DirVector mDefaultClassLibraryDirectories;
+	bool mExcludeDefaultPaths = true;
```

Comment:
true at instatiation time in order to mirror fact that mDefaultClassLibraryDirectories vector is empty.